refined/src/main/scala/main/lucuma/refined.scala

```diff
@@ -84,6 +85,18 @@ object Predicate {
       case _                           => '{ no }
     }
 
+  inline given Predicate[BigDecimal, NonNegative] with
```

Comment:
Thanks! Actually I believe `NonNegative` is an alias for `Not[Negative]`. So I think the instance we are missing is for `Negative`, and then it should compose with the existing instance for `Not` to get us support for `NonNegative`s.

https://github.com/fthomas/refined/blob/d4890dd03ba1755401cd0c32a9bc68931c00f47f/modules/core/shared/src/main/scala-3.0%2B/eu/timepit/refined/numeric.scala#L62

Reply:
Good catch, let me try that.